From 41c6bbad83856551449b6babcf33d4b46a8bda38 Mon Sep 17 00:00:00 2001
From: Matheus Castanho <msc@linux.ibm.com>
Date: Thu, 12 Nov 2020 14:44:31 -0300
Subject: [PATCH] Increase the timeout of test_subprocess

Under heavy load, the subprocess in test_wait_timeout may not complete
in 3s.  Increase that to 10s in order to reduce spurious failures.
---
 Lib/test/test_subprocess.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Lib/test/test_subprocess.py b/Lib/test/test_subprocess.py
index 373542463f..2a0b3f4469 100644
--- a/Lib/test/test_subprocess.py
+++ b/Lib/test/test_subprocess.py
@@ -1522,7 +1522,7 @@ class RunFuncTestCase(BaseTestCase):
                      "time.sleep(3600)"),
                     # Some heavily loaded buildbots (sparc Debian 3.x) require
                     # this much time to start and print.
-                    timeout=3, stdout=subprocess.PIPE)
+                    timeout=10, stdout=subprocess.PIPE)
         self.assertEqual(c.exception.output, b'BDFL')
         # output is aliased to stdout
         self.assertEqual(c.exception.stdout, b'BDFL')
-- 
2.26.2

